Arrowgene.Ddon.GameServer/BazaarManager.cs

```diff
@@ -211,7 +211,8 @@ private List<BazaarExhibition> GetSoldExhibitionsByCharacter(Character character
         private uint calculateProceeds(CDataBazaarItemBaseInfo itemBaseInfo)
         {
             uint totalPrice = itemBaseInfo.Num*itemBaseInfo.Price;
-            return (uint)Math.Clamp(totalPrice - totalPrice*TAXES, 0, uint.MaxValue);
+            uint taxDeduction = (uint)(totalPrice * TAXES);
+            return Math.Clamp(totalPrice - taxDeduction, 0, uint.MaxValue);
```

Comment:
We should probably clamp it to be minimum 1 still, since the UI won't allow the player to claim the proceeds if the amount is 0. If we do a comment should be left explaining this